output.go

```diff
@@ -27,11 +27,15 @@ type OutputAccount struct {
 	Address []byte
 
 	// Nonce is the new account nonce.
-	Nonce *big.Int
+	Nonce uint64
 
-	// Balance is the new account balance.
+	// Balance is the account balance.
 	Balance *big.Int
```

Comment:
do we need this ?

Reply:
It's an awful pain to remove, since some of the tests depend on it and there is too much to change. It did waste me too much time already. Just ignore it for now.

Reply:
ok. will ignore. add comment to ignore for the blockchain.